### PR TITLE
bump version to v1.2.2

### PR DESCRIPTION
**Pull request by lukechampine**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Version History
 ---------------
 
+May 2017:
+
+v1.2.2 (patch release)
+- Fix seed-related bugs affecting wallet balance
+- Gracefully handle missing storage folders
+- Faster + smaller wallet database
```
Comment (DavidVorick) [inline CHANGELOG.md]: can you add a bullet point like "more than 5000 lines of testing, many bugfixes"

I don't know what the actual lines of testing is, but if you have an easy way to count, would be a good stat. It's not a small amount I know.

Comment (lukechampine) [inline CHANGELOG.md]: diff shows 2600 lines of changes since v1.2.1 in files matching `*_test.go`. 3200 lines total. So maybe "2500 lines of new testing"

Comment (DavidVorick) [inline CHANGELOG.md]: cool. That's still a really great number to report.

**Comment by Milkcartonreject** [comment]
When will this be released??

**Pull request merged**